ee/clickhouse/models/group.py

```diff
@@ -1,57 +0,0 @@
-import datetime
```

Comment:
I might be missing something, but since groups are all `ee`, maybe it makes sense to move everything to `ee`, vs. this to `posthog/models` ?